src/views/whototell.njk

```diff
@@ -0,0 +1,64 @@
+{% extends 'layout.njk' %}
+
+{% from 'govuk/components/error-summary/macro.njk' import govukErrorSummary %}
+{% from 'govuk/components/breadcrumbs/macro.njk' import govukBreadcrumbs %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+{% from 'govuk/components/button/macro.njk' import govukButton %}
+{% from "govuk/components/checkboxes/macro.njk" import govukCheckboxes %}
+{% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+
+{% set documentationUrl = 'https://www.gov.uk/government/publications/company-strike-off-dissolution-and-restoration/strike-off-dissolution-and-restoration#how-to-apply-for-strike-off-and-who-to-tell' %}
+
+{% block pageTitle %}
+  {{ serviceName }}
+{% endblock %}
+
+{{ govukBackLink({
+  text: "Back",
+  href: backUri
+}) }}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {{ govukErrorSummary({
+        titleText: "There is a problem",
+        errorList: errors | asGovUKErrorList
+      }) if errors }}
+
+      <h1 class="govuk-heading-xl">Who to tell about the company closing</h1>
+      <p class="govuk-body">Directors must give a copy of this to all interested parties within 7 days of if being submitted.</p>
+      <p class="govuk-body">Read the <a href="{{ documentationUrl }}">full list of parties you must tell about the company closing (opens in a new tab)</a>.</p>
+
+      {{ govukWarningText({
+        text: "Directors can be fined or imprisoned if they do not inform all interested parties.",
+        iconFallbackText: "Warning"
+      }) }}
+
+      <form method="post" action="">
+        {{ govukCheckboxes({
+          idPrefix: "confirmation",
+          name: "confirmation",
+          errorMessage: {
+            text: "You must agree to continue"
```

Comment:
the text should be `errors.confirmation` - otherwise you are duplicating the error message.